### PR TITLE
hdf5-vol-async: fix path needed for h5bench

### PR DESCRIPTION
**Pull request by jeanbez**
Fix the plugin path environment variable needed when using the VOL connector.

### CODE REVIEW

**Review by hyoklee — DISMISSED** [review]

File: var/spack/repos/builtin/packages/hdf5-vol-async/package.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ class Hdf5VolAsync(CMakePackage):
     depends_on("hdf5@1.13: +mpi +threadsafe")
 
     def setup_run_environment(self, env):
-        env.set("HDF5_PLUGIN_PATH", self.spec.prefix)
+        env.set("HDF5_PLUGIN_PATH", join_path(self.spec.prefix, "lib"))
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/hdf5-vol-async/package.py]: Looks good.

Did you know you could simply do the following instead?

```suggestion
        env.set("HDF5_PLUGIN_PATH", self.spec.prefix.lib)
```

**Review by tldahlgren — DISMISSED** [review]
LGTM.  Let me know if you want to shorten the chance as suggested or merge as-is.

File: var/spack/repos/builtin/packages/hdf5-vol-async/package.py (hunk 1 above)
Comment (hyoklee) [reply var/spack/repos/builtin/packages/hdf5-vol-async/package.py]: @jeanbez would you please update your PR following @tldahlgren 's suggestion?

Comment (jeanbez) [reply var/spack/repos/builtin/packages/hdf5-vol-async/package.py]: @hyoklee sure!

**Review by tldahlgren — APPROVED** [review]

**Review by hyoklee — APPROVED** [review]

**Pull request merged**